### ISSUE TITLE
Voltages expander

### ISSUE THREAD

**Issue by martbarras** [TomWhitwell/TuringMachine#158]
Hi 

I’ve just finished the build but the 4th, 6th and 8th leds on the respective sliders don’t light up when the unit is on, these sliders also don’t seem to have any effect either, I’ve checked the solder joints and they all look good so I’m not sure what’s going on, or if there is something I’ve stupidly done.
The unit seems to function ok apart from this 

any help/ pointers greatly appreciated 

Cheers

martin

**Comment by TomWhitwell** [comment]
Hi, sorry to hear that. 
The most likely cause is a bad/missing solder  connection to the ribbon connector or the fader. Or potentially a short to ground at the LEDs? 
You could also test the unit by connecting to the ‘pulses’ output on the Turing - to see if the LEDs flash then. if they do, the fault is in the main module, not the expander.

**Comment by martbarras** [comment]
Thank you Tom 

I appreciate you taking the time to reply, I’ll try these suggestions today and hopefully I’ll be able to fix the issue 🤞

Cheers

Martin

**Comment by martbarras** [comment]
I’ve 
![743EF95C-30B7-4E38-931C-183C95218D30](https://user-images.githubusercontent.com/82546021/115000491-b7c18f80-9e9a-11eb-86ab-60fa834244fc.jpeg)
tried connecting it to the pulses output and only some of the leds flash, not all of them.
 I’ve also reflowed the solder on all the connections and it now seems to have all the leds light up if all the sliders are all over to the right, 5, 6 and 8 however seem to be at a lower brightness and when slid back to the left dim and turn off, the other leds don’t seem to exhibit the same properties. 
 
Is it worth me trying to desolate the sliders and take a look under, the solder points below these all were trimmed carefully and haven’t caused any to be higher up than the rest.

**Comment by TomWhitwell** [comment]
That sounds very odd - the fader positions should have no impact on the led brightness - so that sounds like a short or a bad connection. I wouldn’t advise desoldering the faders unless it’s an absolute last option, as it will be tricky to do, but if you’re confident about everything else... you could try removing the chip, so see if that has any effect on the LEDs (the outputs won’t work while it’s out)

**Comment by martbarras** [comment]
![BFEF6D5C-F411-44B3-8286-243E11DF5642](https://user-images.githubusercontent.com/82546021/115075362-de131980-9ef2-11eb-9624-b3f6a513ad89.jpeg)
![2714A1DC-4C81-49FC-92A8-452EB0E2162C](https://user-images.githubusercontent.com/82546021/115075367-e2d7cd80-9ef2-11eb-91dc-38f693616593.jpeg)
![6ED283CF-063A-4D7D-B0FE-72822A5E4A61](https://user-images.githubusercontent.com/82546021/115075372-e4a19100-9ef2-11eb-8c6b-e2ed9b900e80.jpeg)


Thanks Tom

So tried taking the chip out and led 3,4,6 and 8 don’t light up when they are across on the left.

If I push all these leds to the right they still don’t light up and require 2 of the other ‘working’ leds to be pushed over to the right, before they too will all light up and flash, albeit at a lower brightness than the functioning ones.

**Comment by martbarras** [comment]
After removing the chip and the results above, I decided on a whim to de-solder the power.
Once I had removed all of the solder, I then set about soldering the connector again and low and behold its fixed the issue.

I’m not new to soldering (I used to do as part of my job) and I’ve been checking my work as I go with a 10x mag eye glass. I can’t say I can see any physical difference with the joints than the way they looked before, but just by removing and re applying has solved it.....phew :) 

Exactly as you said Tom, it must have been a bad connection.

Thank you for your time and help, your a legend ! 
Cheers
Martin
